expectations.md

```diff
@@ -1,15 +1,12 @@
-_Jumpstart Live (JSL)_
+_JumpStart Live (JSL)_
 # In-person Student Expectations
 
-## Participating Students
-We expect students participating in Jumpstart Live to not yet be comfortable using loops, arrays, and hashes. We ask our more experienced Ada students to leave this space for those still learning how to use loops, arrays, and hashes effectively, so they can feel confident in the material that you already know before the first day of Ada classes.
-
 ## Punctuality
 We expect all participating students to arrive on time to class except for in cases of emergency. If you do arrive late, please respect the other students learning and take a seat in the back putting forth great effort to reduce distracting other students when you arrive.
 
 ## Prep Work
-* All participating students are expected to complete all prep work for each class. You can find information about the prep work in each days folder
-* We also expect students to come to class with relevant ideas, and questions related to the class topics
+* JumpStart Live follows the inverted classroom model, allowing for more discussion and time for questions in classroom. All participating students are expected to complete all prep work for each class. You can find information about the prep work in each day's folder.
```

Comment:
I don't see a huge benefit of using this term here, they don't know what is was previously so there is really no frame of reference for the inverting. If you want to instead specifically call out the importance of completing the prep work thats fine. Maybe .. _"All participating students are expected to complete all prep work for each class. This will allow more time for discussion and questions during class. You can find information about the prep work in each day's folder."_